com.unity.render-pipelines.core/Editor/CoreEditorUtils.cs

```diff
@@ -99,25 +99,34 @@ public static void DrawFixMeBox(string text, Action action)
 
         // UI Helpers
         /// <summary>Draw a help box with the Fix button.</summary>
-        /// <param name="text">The message text.</param>
-        /// <param name="messageType">The type of the message.</param>
+        /// <param name="message">The message with icon if need.</param>
         /// <param name="action">When the user clicks the button, Unity performs this action.</param>
-        public static void DrawFixMeBox(string text, MessageType messageType, Action action)
+        public static void DrawFixMeBox(GUIContent message, Action action)
```

Comment:
Is this public to users?
If so, then this is a breaking change to public API and we need to deprecate it accordingly.